lib/converter.js

```diff
@@ -88,13 +183,25 @@ async function convert (processor, inputFile, options, timings) {
     if (format) {
       pdfOptions.format = format // Paper format. If set, takes priority over width or height options. Defaults to 'Letter'.
     }
-    const pdf = await page.pdf(pdfOptions)
-    if (outputToStdout) {
-      console.log(pdf.toString())
+    let pdf = debug || await page.pdf(pdfOptions)
+    if (!debug) {
+      if (doc.getAttribute('toc') !== undefined) {
+        pdf = await addOutline(pdf, doc)
+      }
+      fsExtra.writeFileSync(outputFile, pdf)
```

Comment:
I don't think we should write the file on disk when we output to stdout

Reply:
upps. fixed